File: 백준/Bronze/32193. 승강장의 깊이/승강장의 깊이.py
import sys
input = lambda: sys.stdin.readline().rstrip()


def main():
    N = int(input())
    dap = 0
    for _ in range(N):
        A, B = map(int, input().split())
        dap += A
        dap -= B
        print(dap)


if __name__ == '__main__':
    main()

File: 백준/Bronze/32193. 승강장의 깊이/승강장의 깊이.c
#include <stdio.h>
int main() {
    int N, A, B;
    int dap = 0;
    scanf("%d", &N);
    for (int i = 1; i <= N; i ++){
        scanf("%d %d", &A, &B);
        dap += A;
        dap -= B;
        printf("%d\n", dap);
    }
    return 0;
}